### ISSUE TITLE
Suggestion: Add macOS build (to github actions)

### ISSUE THREAD

**Issue by sttng** [stoneface86/trackerboy#12]
Not sure if this is possible at all, but it would certainly be great to get a macOS build (which supports macOS 10.14.6 still).

**Comment by stoneface86** [comment]
It's quite possible, and actually has only been done for v0.5.0. I disabled the macOS builds afterwards since I lack the resources to test them. I could enable them again, but I would leave a disclaimer in the release page that the macOS builds are untested (I develop and test on Linux, and I also test on Windows via a virtual machine). Re-enabling them is simple, just need to uncomment the matrix entry for osx on https://github.com/stoneface86/trackerboy/blob/develop/.github/workflows/build-release.yml#L49

I can try looking into testing them on macOS again, as I was able to get a VM running via QEMU but I'm sure there's a better way. I will also need to research the apple ecosystem more, as I am most experienced with Windows and Linux.

Also, macOS builds are enabled for the nightly builds, but these are debug versions and will run slower compared to the binaries the release workflow generates.
